report/sections/implementation.tex

```diff
@@ -13,15 +13,15 @@ \section{Implementation} \label{impl}
 
 \subsection{kNN}
 
-A kNN classifier simply classifies samples by a majority vote of its neighbors, with a sample being assigned to the class most common among its \texttt{k} nearest neighbors \cite{data-mining-intro}. The hyper-parameter \texttt{n\_neighbors} determines the number of neighbors to use to choose the label of a sample, a total of 15 neighbors attained the best results.
+A kNN classifier simply classifies samples by a majority vote of its neighbors, with a sample being assigned to the class most common among its $k$ nearest neighbors \cite{data-mining-intro}. The hyper-parameter $n_neighbors$ determines the number of neighbors to use to choose the label of a sample, a total of 15 neighbors attained the best results.
```

Comment:
``$n\_neighbors$``, though I still prefer ``\texttt{n\_neighbors}`` as this is not a math formula

Reply:
but they're references to numeric variables in an algorithm

Reply:
[Code listing](https://cs.sharelatex.com/learn/Code_listing)

Reply:
reverting....